src/main/java/net/openhft/chronicle/map/ChronicleMapBuilder.java

```diff
@@ -1607,6 +1608,12 @@ boolean aligned64BitMemoryOperationsAtomic() {
         return this;
     }
 
+    @Override
+    public ChronicleMapBuilder<K, V> skipCloseOnExitHook() {
+        this.skipCloseOnExitHook = true;
+        return null;
```

Comment:
why is it returning null?
And why setter method doesn't accept boolean as a parameter?
```suggestion
        return this;
```

Reply:
I thought the method name makes the intention clear. But I can add it if you prefer the parameter.

Reply:
fixed the null return

Reply:
I'd prefer consistency with other setters - e.g. `B allowSegmentTiering(boolean allowSegmentTiering)` or `B checksumEntries(boolean checksumEntries)`

Reply:
Done